foo.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# coding: utf-8
+
```

Comment:
There is an issue I think. Cant be merged!!
```suggestion
You have to change the code
```

Reply:
Hi